src/Windows/Avalonia.Win32/Interop/UnmanagedMethods.cs

```diff
@@ -1055,12 +1055,17 @@ public enum MONITOR
         }
 
         [StructLayout(LayoutKind.Sequential)]
-        internal class MONITORINFO
+        internal struct MONITORINFO
         {
-            public int cbSize = Marshal.SizeOf<MONITORINFO>();
-            public RECT rcMonitor = new RECT();
-            public RECT rcWork = new RECT();
-            public int dwFlags = 0;
+            public int cbSize;
+            public RECT rcMonitor;
+            public RECT rcWork;
+            public int dwFlags;
+
+            public static MONITORINFO CreateMONITORINFO()
```

Comment:
I meant just `Create` ;) That's what static factory methods are usually called. Is there any reason it needs `MONITORINFO` in there?